src/Imghoard/ImghoardClient.cs

```diff
@@ -91,38 +135,75 @@ public async Task<Uri> PostImageAsync(Stream image, params string[] Tags)
                 await image.CopyToAsync(mStream);
                 bytes = mStream.ToArray();
             }
+            image.Position = 0;
 
-            var imgd = IsSupported(bytes);
+            return await PostImageAsync(bytes, Tags);
+        }
 
-            if (!imgd.Item1)
+        /// <summary>
+        /// Posts a new image to the Imghoard instance
+        /// </summary>
+        /// <param name="bytes">The raw bytes of the image to upload</param>
+        /// <param name="Tags">The tags of the image being uploaded</param>
+        /// <returns>The url of the uploaded image or null on failure</returns>
+        public async Task<string> PostImageAsync(Memory<byte> bytes, params string[] Tags)
+        {
+            if(bytes.Length >= 1000000 && !config.Experimental)
```

Comment:
Preferably do not use 'magic numbers'. Recommend 10 * Mb where Mb is a constant.